stock_move_location/wizard/stock_move_location.xml

```diff
@@ -106,8 +106,14 @@
                                                     name="lot_id"
                                                     groups="stock.group_production_lot"
                                                 />
-                                                <field name="origin_location_id" />
-                                                <field name="destination_location_id" />
+                                                <field
+                                                    name="origin_location_id"
+                                                    force_save="1"
```

Comment:
@dreispt Are you sure this is necessary as force_save should be used if field is readonly

Reply:
I am sure: these values are changed though a form `onchange`, but the changes are not saved because it is read only.
The `force_save` is already there for the list view, but I found it is also needed for the kanban view.